std/stdio.d

```diff
@@ -725,7 +725,8 @@ $(D rawRead) always reads in binary mode on Windows.
     {
         import std.exception : enforce, errnoEnforce;
 
-        enforce(buffer.length, "rawRead must take a non-empty buffer");
+        if (!buffer.length)
+            enforce(false, "rawRead must take a non-empty buffer");
```

Comment:
Oh, I see. Phobos is distributed as a shared library and there is no way to link in statically or to inline calls to Phobos functions. Perhaps std.exception.enforce should be implemented as a mixin so that it can be inlined correctly? There is a ton of code which calls enforce which is not getting inlined.